examples/server_hardware/update_resource.tf

```diff
@@ -0,0 +1,33 @@
+provider "oneview" {
+  ov_username   = var.username
+  ov_password   = var.password
+  ov_endpoint   = var.endpoint
+  ov_sslverify  = var.ssl_enabled
+  ov_apiversion = 3000
+  ov_ifmatch    = "*"
+}
+
```

Comment:
does it allows to update all these attributes under one resource?

Reply:
Updated the code to handle this.